apps/src/music/blockly/blocks/toolboxMode.js

```diff
@@ -0,0 +1,67 @@
+import {BlockTypes} from '../blockTypes';
+import {categoryTypeToLocalizedName} from '../toolbox';
+import {Category} from '../toolbox/types';
+
+const dynamicCategories = [Category.Functions, Category.Variables];
```

Comment:
minor: we have this same list in `toolbox/index.ts` as `dynamicCategories` as well.